library/lua/gui/widgets.lua

```diff
@@ -1499,17 +1501,16 @@ CycleHotkeyLabel.ATTRS{
 function CycleHotkeyLabel:init()
     self:setOption(self.initial_option)
 
-    local val_gap = 1
     if self.label_below then
-        val_gap = 0 + (self.key_back and 1 or 0) + (self.key and 3 or 0)
+        self.val_gap = 0 + (self.key_back and 1 or 0) + (self.key and 3 or 0)
```

Comment:
perhaps mention that the `val_gap` value is ignored if `label_below` is `true`

Reply:
Good idea.

I wonder if we should somehow make `val_gap` affect the label's position even when `label_below` is true. 🤔 
for now i'll just update the documentation.